source/agora/test/QuorumPreimage.d

```diff
@@ -179,8 +179,8 @@ unittest
         // 1
         QuorumConfig(6, [
             WK.Keys.NODE3.address,
-            WK.Keys.NODE2.address,
             WK.Keys.NODE6.address,
+            WK.Keys.NODE5.address,
```

Comment:
Wait why is this test changing ?

Reply:
As the order of the Genesis validator keys changed then the nodes in the unit test changed.

Reply:
Ah, the test themselves didn't change but the order in which we get them did. Gotcha.